set.mm

```diff
@@ -666258,6 +666624,670 @@ is real and it is less than or equal to its inferior limit (in such a
+      KXL $.
+  $}
+
+  ${
+    fuzxrpmcn.1 $e |- Z = ( ZZ>= ` M ) $.
+    fuzxrpmcn.2 $e |- ( ph -> F : Z --> RR* ) $.
+    $( A function mapping from an upper set of integers to the extended reals
+       is a partial map on the complex numbers.  (Contributed by Glauco
+       Siliprandi, 5-Feb-2022.) $)
+    fuzxrpmcn $p |- ( ph -> F e. ( RR* ^pm CC ) ) $=
+      ( cc cxr cvv wcel cnex a1i xrex wss uzsscn2 fpmd ) AGHDBIIGIJAKLHIJAMLDGN
+      ACDEOLFP $.
+  $}
+
+  ${
+    $d F j k u x $.  $d M j $.  $d Z j k $.  $d ph u x $.
+    xlimpnflemim2.k $e |- F/ k ph $.
+    xlimpnflemim2.j $e |- F/ j ph $.
+    xlimpnflemim2.m $e |- ( ph -> M e. ZZ ) $.
+    xlimpnflemim2.z $e |- Z = ( ZZ>= ` M ) $.
+    xlimpnflemim2.f $e |- ( ph -> F : Z --> RR* ) $.
+    xlimpnflemim2.g $e |- ( ph -> A. x e. RR E. j e. Z A. k e. ( ZZ>= ` j ) x <
+       ( F ` k ) ) $.
+    $( The "if" part of the biconditional in ~ xlimpnf .  (Contributed by
+       Glauco Siliprandi, 5-Feb-2022.) $)
+    xlimpnflemim2 $p |- ( ph -> F ~~>* +oo ) $=
+      ( cpnf cfv cxr wcel wa nfv nfan vu cle cordt clm wbr clsxlim cc cpm co cv
+      cdm cuz wral cz wrex wi fuzxrpmcn pnfxr a1i wss cr pnfnei adantll clt w3a
+      cioc 3adant1 wceq fdmd 3ad2ant1 eleqtrrd ad5ant134 adantl4r simp-4r rexrd
+      uztrn2 simp-4l ad4ant23 ffvelrnda syl2anc simpr wf ffvelrnd pnfged eliocd
+      ad5ant1345 sseldd jca ex ralimda 3impia r19.21bi adantr reximdd wb rexuz3
+      syl ad2antrr mpbid rexlimdva2 mpd ralrimiva nfcv ctopon letopon mpbir3and
+      lmbr3 df-xlim breqi sylibr ) AENUBUCOZUDOZUEZENUFUEAXMEPUGUHUIQNPQZNUAUJZ
+      QZDUJZEUKZQZXQEOZXOQZRZDCUJZULOZUMZCUNUOZUPZUAXKUMAEFGKLUQXNAURUSAYGUAXKA
+      XOXKQZRZXPYFYIXPRBUJZNVFUIZXOUTZBVAUOZYFYHXPYMABXOVBVCAYMYFUPYHXPAYLYFBVA
+      AYJVAQZRZYLRZYECGUOZYFYPYJXTVDUEZDYDUMZYECGYOYLCAYNCIYNCSTYLCSTYPYCGQZYSY
+      EYPYTRZYRYBDYDYPYTDYOYLDAYNDHYNDSTYLDSTYTDSTUUAXQYDQZRZYRYBUUCYRRZXSYAAYN
+      YLYTUUBYRXSAYTUUBXSYLYRAYTUUBVEZXQGXRYTUUBXQGQZAFXQYCGKVPZVGZAYTXRGVHUUBA
+      GPELVIVJVKVLVMUUDYKXOXTYOYLYTUUBYRVNYOYTUUBYRXTYKQYLYOYTRUUBRZYRRZYJNXTUU
+      JYJAYNYTUUBYRVNVOXNUUJURUSUUJAUUFXTPQAYNYTUUBYRVQYTUUBUUFYOYRUUGVRAGPXQEL
+      VSVTUUIYRWAAYTUUBXTNUBUEYNYRUUEXTUUEGPXQEAYTGPEWBUUBLVJUUHWCWDVLWEWFWGWHW
+      IWJWKYOYSCGUOZYLAUUKBVAMWLWMWNAYQYFWOZYNYLAFUNQUULJYBCDFGKWPWQWRWSWTWRXAW
+      IXBAUANCDEXKPDEXCXKPXDOQAXEUSXGXFENUFXLXHXIXJ $.
+  $}
+
+  ${
+    $d A w y $.  $d A x y $.  $d B w y $.  $d C x $.  $d D w $.  $d X x y $.
+    $d ph w y $.
+    cnrefiisplem.a $e |- ( ph -> A e. CC ) $.
+    cnrefiisplem.n $e |- ( ph -> -. A e. RR ) $.
+    cnrefiisplem.b $e |- ( ph -> B e. Fin ) $.
+    cnrefiisplem.c $e |- C = ( RR u. B ) $.
+    cnrefiisplem.d $e |- D = ( { ( abs ` ( Im ` A ) ) } u. U_ y e. ( ( B i^i CC
+       ) \ { A } ) { ( abs ` ( y - A ) ) } ) $.
+    cnrefiisplem.x $e |- X = inf ( D , RR* , < ) $.
+    $( Lemma for ~ cnrefiisp (some local definitions are used) (Contributed by
```

Comment:
final period

Reply:
Fixed in the latest commit

Reply:
I fixed it in the latest commit (I hope, I had to search manually). If it's a convention, it'd be nice to have it automated (checked / fixed)

Reply:
It's more like a convention of the English language. In this case, it would be doable in principle (check that `(Contributed by` is preceeded by `.  `), but I'm not sure this is worth it. (@digama0 ?)